addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
Either we do it right or we don't do it at all. There is no in between. If you continue to call this a stupid change, we will have a hard time, I promise you.